src/training/finetune.py

```diff
@@ -0,0 +1,234 @@
+import argparse
```

Comment:
This is specific to FLAVA. We probably shouldn't land this file into main until it is more generic.